.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1031 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
but file's still here. you'd need to explicitly delete it in a new commit now.

---
In reply to: [73150150](https://github.com/Azure/azure-iot-protocol-gateway/pull/58#discussion_r73150150) [](ancestors = 73150150)